recipes/rcssmin/run_test.py

```diff
@@ -0,0 +1 @@
+import rcssmin
```

Comment:
Can you drop this file and add the import to a test section in the `meta.yaml`:

https://conda.io/docs/building/meta-yaml.html#python-imports